simulation/revolve2/simulation/scene/geometry/textures/_texture.py

```diff
@@ -0,0 +1,40 @@
+from dataclasses import dataclass
+
+from ..._color import Color
+from ._map_type import MapType
+
+
+@dataclass(frozen=True, kw_only=True)
+class Texture:
+    """An abstract texture for geometric models."""
+
+    builtin: str
+    """
+    builtin can be used to reference specific builtin textures in the target simulator.
+    
+    Mujoco has the following builtin textures available: 
```

Comment:
again, we cannot mention mujoco. this is a revolve2 abstraction. any user using this should be able to use it with zero knowledge of the existance of mujoco.

Reply:
In my opinion, if these are textures built into Revolve2, that means there is a limited number of textures. When there is a limited number of choices, an enum improves usability